modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/AssignModal.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayModal from '@clayui/modal';
+import ClayTabs from '@clayui/tabs';
+import React, {useState} from 'react';
+
+import {TProperty} from '../components/PropertiesTable';
+import ChannelTab from './ChannelTab';
+import SitesTab from './SitesTab';
+
+interface IAssignModal {
```

Comment:
Rename `IAssignModal` to `IAssignModalProps`

Reply:
Fixed